projects/skia/svg_dom.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+max_len = 30000
```

Comment:
This needs to be checked in the fuzzer as otherwise wont work for other fuzzing engines.
if (size > 30000)
  return 0;